### PR TITLE
[ecosystem] Create IT1 stats table.

### PR DESCRIPTION
**Pull request by zacharydenton**
## Motivation

https://user-images.githubusercontent.com/310773/169176205-f7e8e500-2880-440e-8a92-f96e90c502fb.mp4

## Test Plan

Manual

## Related PRs

(If this PR adds or changes functionality, please take some time to update the docs at https://github.com/aptos-labs/aptos-core/tree/main/developer-docs-site, and link to your PR here.)

### CODE REVIEW

File: ecosystem/platform/server/app/helpers/application_helper.rb [hunk 1]
```diff
@@ -4,4 +4,13 @@
 # SPDX-License-Identifier: Apache-2.0
 
 module ApplicationHelper
+  def parse_sort(params)
+    result = []
+    sort_order = { '+' => 1, '-' => -1 }
```
Comment (CapCap) [inline ecosystem/platform/server/app/helpers/application_helper.rb:9]: this can be a class constant

Comment (zacharydenton) [reply ecosystem/platform/server/app/helpers/application_helper.rb]: Done.

File: ecosystem/platform/server/app/helpers/application_helper.rb [hunk 2]
```diff
@@ -4,4 +4,13 @@
...
+    params.fetch(:sort, '').split(',').map do |attr|
```
[the remark this answers is not in the document]
Comment (zacharydenton) [reply ecosystem/platform/server/app/helpers/application_helper.rb:10]: Done.

**Comment by davidiw** [comment]
sorting is good for you! looks nice

**Review by CapCap — APPROVED** [review]

**Pull request closed without merging**